xml/adm_sudo.xml

```diff
@@ -0,0 +1,611 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.adm.sudo"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>sudo <!-- TODO: Better title --></title>
+  <subtitle><quote>With great power comes great responsibility</quote></subtitle>
+  <info>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:bugtracker></dm:bugtracker>
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+  </info>
+
+  <para>
+    Many commands and system utilities need to be run as &rootuser;
+    to modify files and/or perform tasks that only the super user is allowed to.
+
+    For security purposes and to avoid running dangerous commands accidentially,
```

Comment:
For security _reasons_ and to avoid _accidentally_ running dangerous commands...